src/components/productCard/ProductCard.tsx

```diff
@@ -1,14 +1,21 @@
 import React from "react";
-import PropTypes from "prop-types";
 import "./ProductCard.css";
 
+interface ProductCardProps {
+  title: string;
+  image: string;
+  price: number;
+  description: string;
+  descriptionBackgroundColor: string;
```

Comment:
@N-i-z Since you have created classes for both men's & women's variants with their colors in it, let's turn this into a `category` props! Pick your classNames based on that category prop....